lib/ViewModels/GazetteerSearchProviderViewModel.js

```diff
@@ -113,6 +114,24 @@ GazetteerSearchProviderViewModel.prototype.search = function(searchText) {
     });
 };
 
+// Given a list of results sorted in decreasing importance, strip results that are close to another result with the same name
+function stripDuplicates(results) {
+    var i;
+    var placeshash={};
+    var stripped = [];
+    for (i=0; i < results.length; i++) {
+        var lat = Number(results[i].location.split(',')[0]).toFixed(1);
```

Comment:
`toFixed(1)` means places that are actually pretty far apart will be considered the same place, right?